backend/src/main/java/com/bakdata/conquery/resources/admin/ui/DatasetsUIResource.java

```diff
@@ -132,6 +134,7 @@ public static class DatasetInfos {
 
 		private Dataset ds;
 		private Collection<SecondaryIdDescription> secondaryIds;
+		private Collection<InternToExternMapper> internToExternMappers;
```

Comment:
du hattest einen screenshot angehängt für AdminUI, wo machst du das?

Reply:
Das ist in dem FTL-Template